backend/src/test/java/com/woowacourse/levellog/acceptance/ConcurrentAcceptanceTest.java

```diff
@@ -0,0 +1,172 @@
+package com.woowacourse.levellog.acceptance;
+
+import static com.woowacourse.levellog.fixture.MemberFixture.EVE;
+import static com.woowacourse.levellog.fixture.MemberFixture.PEPPER;
+import static com.woowacourse.levellog.fixture.RestAssuredTemplate.get;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import com.woowacourse.levellog.feedback.dto.response.FeedbackResponses;
+import com.woowacourse.levellog.fixture.ConcurrentRequester;
+import com.woowacourse.levellog.interviewquestion.dto.query.InterviewQuestionSearchQueryResult;
+import com.woowacourse.levellog.interviewquestion.dto.query.InterviewQuestionSearchQueryResults;
+import com.woowacourse.levellog.member.dto.response.MemberResponses;
+import com.woowacourse.levellog.team.dto.response.TeamDetailResponse;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("동시성 관련 기능")
+class ConcurrentAcceptanceTest extends AcceptanceTest {
+
+    /*
+     * Scenario: 한 명의 사용자가 동시에 회원가입
+     *   when: 동시에 여러 개의 피드백 작성을 요청한다.
+     *   then: 한 개의 피드백만 작성에 성공한다.
+     */
+    @Test
+    @DisplayName("멤버 동시 가입")
+    void concurrentlyCreateMember() throws InterruptedException {
+        // given
+        final int threadPoolSize = 2;
+        final ConcurrentRequester concurrentRequester = new ConcurrentRequester(threadPoolSize);
+
+        // when
+        for (int i = 0; i < threadPoolSize; i++) {
+            concurrentRequester.execute(PEPPER::login);
+        }
+        concurrentRequester.await();
```

Comment:
```suggestion
        // given
        final int requestCount = 2;
        final ConcurrentRequester concurrentRequester = new ConcurrentRequester();

        // when
        concurrentRequester.execute(PEPPER::login, requestCount);
```

`ConcurrentRequester.execute` 내부에서 `countDownLatch.await()`까지 실행하면 어떨까요? 반복문 코드도 내부에 감춘다면 사용성이 더 좋을 것 같다고 생각해요 🥺